plugins/logitech-hidpp/fu-logitech-hidpp-runtime-bolt.c

```diff
@@ -51,17 +51,8 @@ fu_logitech_hidpp_runtime_bolt_detach(FuDevice *device, FuProgress *progress, GE
 	}
 
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
-	/* generate a message if not already set */
-	if (fu_device_get_update_message(device) == NULL) {
-		g_autofree gchar *str = NULL;
-		str = g_strdup_printf("%s needs to be manually restarted to complete the update. "
-				      "Please unplug it and plug it back again.",
-				      fu_device_get_name(device));
-		fu_device_set_update_message(device, str);
-	}
 	fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_IMMEDIATE);
 	fwupd_request_set_id(request, FWUPD_REQUEST_ID_REMOVE_REPLUG);
-	fwupd_request_set_message(request, fu_device_get_update_message(device));
```

Comment:
don't we need this? I did kinda assume that we'd construct the gchar *str and then set it as the request message.

Reply:
My bolt receivers on my desk need replugging, so we probably need a way to ask the device "do you need a replug" -- I had assumed that logitech would have different VID/PIDs for the different behaviour. Or am I completely mistaken and all "available in the shops" bolt receivers will no longer need a replug? If that's the case then we need to remove the fwupd_request_set_id() and fwupd_request_set_kind() lines too.